src/stylesheets/elements/_inputs.scss

```diff
@@ -77,8 +77,19 @@ select {
   label {
     margin-top: 0;
   }
+
+  .usa-input-inline {
+    border: $input-border-width solid $color-gray;
+    width: inherit;
+  }
+
+  .usa-input-inline-error {
+    border: 3px solid $color-secondary-dark;
```

Comment:
@jasonkaedingICE thank you for pointing this out to us and expressing the need for better examples. I am going to take this information to an upcoming workshop where we are determining some next steps for design and research. In the meantime, check out the guidance [found here](https://standards.usa.gov/components/form-controls/#text-input) to learn more about various states available for our form controls.